src/Generator.Bind/LicenseResource.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using System.Reflection;
+
+namespace Bind
+{
+    /// <summary>
+    /// Class for handling embedded license text.
+    /// </summary>
+    internal class LicenseResource
+    {
+        private static string _license;
+
+        /// <summary>
+        /// Gets and returns the license text from the embedded resource.
+        /// </summary>
+        public static string LicenseText
+        {
+            get
+            {
+                if (_license != null)
+                {
+                    return _license;
+                }
+
+                var streamReader = new StreamReader(Assembly.GetAssembly(typeof(Program)).GetManifestResourceStream("Specifications/License.txt"));
```

Comment:
This path won't work - use a dot instead of a slash.

Reply:
Actually it won't work at all. You need to explicity state the full namespace (i.e. Bind.Specifications.License.txt)